Classes/Hooks/Interfaces/RegistrationListCsv.php

```diff
@@ -17,9 +19,9 @@ interface RegistrationListCsv extends Hook
      * This allows modifying the complete CSV text right before it is delivered.
      *
      * @param string $csv the CSV text produced by `AbstractRegistrationListView::render()`
-     * @param \Tx_Seminars_Csv_AbstractRegistrationListView $registrationList the CSV data provider
+     * @param AbstractRegistrationListView $registrationList the CSV data provider
      *
      * @return string the modified CSV text to use
      */
-    public function modifyCsv(string $csv, \Tx_Seminars_Csv_AbstractRegistrationListView $registrationList): string;
+    public function modifyCsv(string $csv, AbstractRegistrationListView $registrationList): string;
```

Comment:
Oh, a hook. Then we cannot namespace the `Tx_Seminars_Csv_AbstractRegistrationListView` and move namespacing this class to the 4.0 milestone so 3.4 won't be breaking.